docs/guides/theme_architecture/blocks/schema/variables/variable_types/select.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Select
+parent: Variable types
+---
+
+# Select
+
+Renders a select input and returns the value of the selected option.
+Takes an `options` parameter which defines the options in the select, this is an array of label, value pairs.
+
+##### syntax
+{% raw %}
+```
+---
+my_variable:
+    type: select
+    default: opt_1
```

Comment:
If no default there's a blank first option?

Reply:
Added the same details as radio (and double checked) - it will select the first option by default